specification/storageimportexport/resource-manager/Microsoft.ImportExport/2016-11-01/storageimportexport.json

```diff
@@ -1,1210 +1,1138 @@
+                            "description": "The phone number for the Azure data center. ",
+                            "type": "string"
+                        },
+                        "supportedCarriers":{
+                            "type": "array",
+                            "description": "A list of carriers that are supported at this location. ",
+                            "items":{
+                                "type": "string"
+                            }
+                        },
+                        "alternateLocations":{
+                            "type": "array",
+                            "description": "A list of location IDs that should be used to ship shipping drives to for jobs created against the current location. If the current location is active, it will be part of the list. If it is temporarily closed due to maintenance, this list may contain other locations. ",
+                            "items":{
+                                "type": "string"
+                            }
+                        }
+                    }
+                }
+            }
         },
-        "cancelRequested": {
-          "type": "boolean",
-          "description": "Indicates whether a request has been submitted to cancel the job."
+        "ReturnAddress": {
+            "description": "Specifies the return address information for the job.",
+            "required": [
+              "recipientName",
+              "streetAddress1",
+              "city",
+              "postalCode",
+              "countryOrRegion",
+              "phone",
+              "email"
+            ],
+            "properties": {
+                "recipientName": {
+                    "type": "string",
+                    "description": "The name of the recipient who will receive the hard drives when they are returned. "
+                },
+                "streetAddress1": {
+                    "type": "string",
+                    "description": "The first line of the street address to use when returning the drives. "
+                },
+                "streetAddress2": {
+                    "type": "string",
+                    "description": "The second line of the street address to use when returning the drives. "
+                },
+                "city": {
+                    "type": "string",
+                    "description": "The city name to use when returning the drives."
+                },
+                "stateOrProvince": {
+                    "type": "string",
+                    "description": "The state or province to use when returning the drives."
+                },
+                "postalCode": {
+                    "type": "string",
+                    "description": "The postal code to use when returning the drives."
+                },
+                "countryOrRegion": {
+                    "type": "string",
+                    "description": "The c
```

Comment:
@leoz-ms just to make sure, is this one supposed to be a client parameter instead of a method parameter? :)

Reply:
ahhh of course it shows in the wrong spot... I mean the `Accept-Language` parameter that besides `subscriptionId` (where it makes perfect sense) *may* have been left as a client parameter by accident :)

Reply:
I don't expect Accept-Language to change across requests just like what subscriptionId, so it makes sense to leave it as client parameter. BTW, I don't find other swaggers defining it, really surprising.